index.html

```diff
@@ -6,8 +6,58 @@
     <title>Tour and Travel</title>
     <link rel="stylesheet" href="styles/index.css" />
     <link rel="stylesheet" href="styles/shared.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />

```

Comment:
bunları silmelisin. Bunları shared.css'in içerisinde importluyoruz.

Reply:
Düzeltildi.